util-core/src/main/scala-2.13+/twitter/util/HealthyQueue.scala

```diff
@@ -0,0 +1,42 @@
+package com.twitter.util

```

Comment:
we will need to find out if this also works for pants build.

Reply:
I'll see if I can get a Pants setup working, and read up about how it selects which files to build for which targets.

 Would you happen to know whether it works on WSL?

Reply:
Pants confirmed we can do similar things as sbt to point to the 213 directories, I think we can do it internally when merging this.

Reply:
I just got the opposite answer from the pants slack from stu hood, who suggested an issue against pants, and different branches as a workaround, so that's a relief

Reply:
Yes, that won't block this is what I heart. so 👍 

>Would you happen to know whether it works on WSL?
What is WSL?

Reply:
WSL is the Linux that comes with Windows, so I can run Pants locally without spinning up a VM, but if I can make the whole pants thing someone else's problem for the extent of this PR, then I'm all good.